src/gluonts/transform/feature.py

```diff
@@ -30,6 +30,129 @@ def target_transformation_length(
     return target.shape[-1] + (0 if is_train else pred_length)
 
 
+class MissingValueImputation:
+    """
+    The parent class for all the missing value imputation classes.
+    You can just implement your own inheriting this class.
+    """
+
+    @validated()
+    def __init__(self) -> None:
+        pass
+
+    def __call__(self, values: np.ndarray) -> np.ndarray:
+        raise NotImplementedError()
+
+
+class LeavesMissingValues(MissingValueImputation):
+    def __call__(self, values: np.ndarray) -> np.ndarray:
+        return values
+
+
+class DummyValueImputation(MissingValueImputation):
+    @validated()
+    def __init__(self, dummy_value: float = 0.0) -> None:
+        self.dummy_value = dummy_value
+
+    def __call__(self, values: np.ndarray) -> np.ndarray:
+        nan_indices = np.where(np.isnan(values))
+        values[nan_indices] = self.dummy_value
+        return values
+
+
+class MeanValueImputation(MissingValueImputation):
+    """
+    Careful this is not a 'causal' method in the sense that it leaks information about the furture in the imputation.
+    You may prefer to use CausalMeanValueImputation instead.
+    """
```

Comment:
It would be nice to have a description of what these classes do: maybe you could have a docstring here, for each classes, clarifying what the __call__ method does